common4j/src/main/com/microsoft/identity/common/java/crypto/AsymmetricAlgorithm.java

```diff
@@ -20,36 +20,28 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
```

Comment:
This file has a **WEIRD** diff.  I'm not sure that it won't cause problems down the line.

Reply:
I killed this class... somehow git thought it's close enough to AsymmetricAlgorithm.... probably because of the license?